launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -1040,6 +1040,10 @@ QString MinecraftInstance::getStatusbarDescription()
                     .arg(Time::prettifyDuration(totalTimePlayed(), APPLICATION->settings()->get("ShowGameTimeWithoutDays").toBool())));
         }
     }
+    auto currentShortcuts = shortcuts();
+    if (!currentShortcuts.isEmpty()) {
+        description.append(tr(", %n shortcut(s) registered", "", currentShortcuts.size()));
```

Comment:
This may seem usefull but the overload of info on this line is exceeding

Reply:
Yeah, after thinking about this more I think it is more appropriate to add this to the deletion confirmation screen instead